resources/public/api/conf/1.0/application.yaml

```diff
@@ -204,22 +195,104 @@ paths:
             application/json:
               schema:
                 allOf:
-                - $ref: '#/components/schemas/individualPayeTestSupportSaTaxAnnualSummaryResponse'
-              example:
-                pensionsAnnuitiesAndOtherStateBenefits:
-                  otherPensionsAndRetirementAnnuities: 36.5
-                  incapacityBenefit: 980.45
-                refunds:
-                  taxRefundedOrSetOff: 325
-                employments:
-                - employerPayeReference: 123/AB456
-                  taxTakenOffPay: 890.35
-                - employerPayeReference: 456/AB456
-                  taxTakenOffPay: 224.99
+                  - $ref: '#/components/schemas/individualPayeTestSupportSaBenefitsAnnualSummaryResponse'
```

Comment:
This is show the schema for benefits instead of tax. Please update the schema to `individualPayeTestSupportSaTaxAnnualSummaryResponse`